apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -151,6 +154,20 @@ public ApolloStore apolloStore() {
     return apolloStore;
   }
 
+  /**
+   * Sets the idleCallback which will be called when this ApolloClient is idle.
+   */
+  public void setIdleCallback(Runnable idleCallback) {
```

Comment:
Remove prefix `set`
I would make callback to be interface `interface IdelCallback` instead of generic `Runnable`